packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -55,13 +55,28 @@ const addWebRender = (content, authProvider) => {
   const redwoodProviderLines = redwoodProvider.split('\n').map((line) => {
     return '  ' + line
   })
+  const customRenderOpen = authProvider.render
+    ? authProvider.render.reduce(
+        (acc, component) => acc + indent + `<${component}>`,
+        ''
+      )
+    : ''
```

Comment:
You obviously don't have to, since this is all personal preference, but if you wanted to you could do this

```suggestion
  const customRenderOpen = (authProvider.render || []).reduce(
    (acc, component) => acc + indent + `<${component}>`,
    ''
  )
```

Reply:
That looks clean. Changed!